retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
Whatever we do we'll need plenty of docs, however I'm not worried about users being unable to create a basic setup as they'll primarily be copying and pasting.

> Then we also needed to make that configurable, as we did for the Pushgateway.

I wonder if we do, as the only labels that are special in terms of scraping are `job` and `instance`.